rollbar/src/main/java/com/rollbar/Rollbar.java

```diff
@@ -984,4 +986,12 @@ protected void setTransform(PayloadTransform transform) {
     public Rollbar transform(PayloadTransform transform) {
         return new Rollbar(accessToken, environment, sender, codeVersion, platform, language, framework, context, request, person, server, custom, notifier, responseHandler, filter, transform);
     }
+
+    private LinkedHashMap<String, Object> wrapCustom(Map<String, Object> custom) {
+        return new LinkedHashMap<String, Object>(custom);
+    }
+
+    private LinkedHashMap<String, Object> defaultCustom() {
+        return new LinkedHashMap<String, Object>();
```

Comment:
In this case the 'defaultCustom' would actually need to be null, which would cause it to be skipped in the serializer.